src/config/timePeriods/initialLocking.js

```diff
@@ -2,8 +2,27 @@ const dateUtil = require('../../helpers/dateUtil')
 
 // The lock peridod has some default config, but it ca be modified with
 // env variables
-const LOCK_PERIOD_START = '2018-11-01 00:00'
-const LOCK_PERIOD_END = '2019-01-31 23:59:59'
+const now = new Date()
```

Comment:
This was convenient for the deployment in production. The less config we have to pass when we deploy this the better. Anyways, if it's in the middle of the way for the tests, we can have a similar logic, but not done in the config file.

We can declare a add function in date util, and use it like this:

```js
let LOCK_PERIOD_START, LOCK_PERIOD_END
if (!process.env.LOCK_PERIOD_START && !process.env.LOCK_PERIOD_END) {
  // Default dates (+1 month, +3 month)
  const now = new Date()
  LOCK_PERIOD_START = dateUtil.add(now, 1, 'month')
  LOCK_PERIOD_END = dateUtil.add(now, 3, 'month')
} else if (process.env.LOCK_PERIOD_START && process.env.LOCK_PERIOD_END) {
  // Provided dates
  LOCK_PERIOD_START = process.env.LOCK_PERIOD_START
  LOCK_PERIOD_END =  process.env.LOCK_PERIOD_END
} else {
  throw new Error('bla bla')
}
```

Reply:
There's a function in moment that does this exactly. We already use moment in dateUtil

Reply:
Done